src/scripts/migrateTenantOrgData.js

```
@@ -0,0 +1,2433 @@
+	const sourceRolesFromUsers = await querySelect(
+		sequelize,
+		`SELECT DISTINCT unnest(roles) AS role_id
+		 FROM users
+		 WHERE tenant_code = $sourceTenant
+		   AND id = ANY($userIds);`,
+		{ sourceTenant, userIds },
+		tx
+	)
+
+	const requiredRoleIds = new Set()
+	sourceRolesFromUor.forEach((row) => row.role_id !== null && requiredRoleIds.add(Number(row.role_id)))
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)